### ISSUE TITLE
Download Competition Data

### ISSUE THREAD

**Issue by btcain4** [btcain4/leash-bio-ai#6]
As a developer, 
I want to download the [competition dataset](https://www.kaggle.com/competitions/leash-BELKA/data?select=sample_submission.csv) into my available local HDD space, 
So that I can start determining processes of working with this data on local compute and eventually conducting analysis. 

Acceptance Criteria:

- [x] Determine on .csv or .parquet download format
- [x] Download data to HDD location
- [x] Name data according to standard naming conventions

**Comment by btcain4** [comment]
This issue has been resolved in the following manner:

- Training set saved as .parquet in HDD
- Test set saved as .parquet in HDD
- Sample submission saved as .csv in HDD
